### DISCUSSION TITLE
Error migration database

### DISCUSSION THREAD

**Discussion by mobiusrpi** [eusonlito/GPS-Tracker#66] [Q&A]
Hi
I follow your guideline to install on local server GPS Tracker
On step 7, I have the following error message

The command "mysql  --user="${:LARAVEL_LOAD_USER}" --password="${:LARAVEL_L  
  OAD_PASSWORD}" --host="${:LARAVEL_LOAD_HOST}" --port="${:LARAVEL_LOAD_PORT}  
  " --database="${:LARAVEL_LOAD_DATABASE}" < "${:LARAVEL_LOAD_PATH}"" failed.  
                                                                               
  Exit Code: 1(General error)                                                  
                                                                               
  Working directory: /var/www/GPS-Tracker                                      
                                                                               
  Output:                                                                      
  ================                                                                                            
                                                                               
  Error Output:                                                                
  ================                                                             
  ERROR 1901 (HY000) at line 30: Function or expression '`st_latitude`()' can  
  not be used in the GENERATED ALWAYS AS clause of `latitude`   

No table created in database gpstracker, only alarm item, appear

Thank for your help

**Comment by eusonlito** [comment]
Are you using MySQL 8.0.12 or higher?

**Reply by mobiusrpi** [reply]
I use Mysql 8.3

**Comment by mobiusrpi** [comment]
Sorry I upgraded my server and the database is mariadb 10.11.6

    Serveur : Localhost via UNIX socket
    Type de serveur : MariaDB
    Connexion au serveur : SSL n'est pas utilisé Documentation
    Version du serveur : 10.11.6-MariaDB-0+deb12u1 - Debian 12
    Version du protocole : 10
    Utilisateur : root@localhost
    Jeu de caractères du serveur : UTF-8 Unicode (utf8mb4)

**Reply by eusonlito** [reply]
Sorry but MariaDB is not supported.

**Reply by adarsh23590** [reply]
then which is supported i am using XAMPP

**Comment by mobiusrpi** [comment]
Hi

Thank you for your help
I replaced the database Mariadb by mysql 8.0.37
I finished the installation
I attached my configuration file for test as localhost

When I run in my navigator I have a message error  : 500 Internal server
error

In the error.log; the error is located in .htaccess file

/var/www/GPS-Tracker/public/.htaccess: Invalid command 'Header', perhaps
misspelled or defined by a module not included in the server configuration

Do you have an idea ?

Joel

**Reply by eusonlito** [reply]
You need to enable Apache `headers` module.

**Comment by adarsh23590** [comment]
> Hi I follow your guideline to install on local server GPS Tracker On step 7, I have the following error message
> 
> The command "mysql --user="${:LARAVEL_LOAD_USER}" --password="${:LARAVEL_L OAD_PASSWORD}" --host="${:LARAVEL_LOAD_HOST}" --port="${:LARAVEL_LOAD_PORT} " --database="${:LARAVEL_LOAD_DATABASE}" < "${:LARAVEL_LOAD_PATH}"" failed.
> 
> Exit Code: 1(General error)
> 
> Working directory: /var/www/GPS-Tracker
> 
> # Output:
> # Error Output:
> ERROR 1901 (HY000) at line 30: Function or expression '`st_latitude`()' can not be used in the GENERATED ALWAYS AS clause of `latitude`
> 
> No table created in database gpstracker, only alarm item, appear
> 
> Thank for your help

I also get same error. I am using Xampp and php version 8.2. any solution?

**Reply by eusonlito** [reply]
Are you using MySQL 8.0.12 or higher?

**Reply by adarsh23590** [reply]
I am using XAMPP 
![image](https://github.com/eusonlito/GPS-Tracker/assets/171769650/9c0ee96f-db3d-4688-9ef6-2535af79db1a)
![image](https://github.com/eusonlito/GPS-Tracker/assets/171769650/65304200-4ed9-4e6b-8987-ec918bd63f2d)

 please check

**Reply by eusonlito** [reply]
MariaDB does not support the required GIS functionalities.

**Reply by adarsh23590** [reply]
I have changes to mysql from mariadb but stilll getting error
![image](https://github.com/eusonlito/GPS-Tracker/assets/171769650/3696a434-cbb6-4c2f-a371-eaf7e6a0c66d)
![image](https://github.com/eusonlito/GPS-Tracker/assets/171769650/3f11f750-15f6-4a87-94ac-153c6d545096)
my environment variables are already set but still getting error

**Reply by eusonlito** [reply]
Sorry, windows is not supported by the requirements to be able to run the protocol servers.
